stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
I think, after patch, single ```os(Windows)``` (equivalent ```env(msvc) || env(cygnus)```, or current ```os(Windows) || os(Cygwin)```) will be rarely occurred, since the environments ```msvc``` and ```cygnus``` are very different in the compiler and in their main libraries.

In programmer perspective, the codes written for ```env(cygnus)``` will be more similar to the codes for Linux/FreeBSD/.. than Windows(MSVC).